eng/targets/Helix.props

```diff
@@ -26,6 +25,14 @@
     <TestDependsOnAspNetRuntime>true</TestDependsOnAspNetRuntime>
   </PropertyGroup>
 
+  <!-- Override a few properties in innermost (Publish) build for Helix submission. -->
+  <PropertyGroup Condition=" '$(HelixTargetQueue)' != '' ">
+    <HelixTimeout Condition="$(HelixTargetQueue.StartsWith('Windows.10.Amd64'))">00:40:00</HelixTimeout>
```

Comment:
we should consider getting rid of this difference between queues and just use 45 min universal timeouts

Reply:
Could you explain why❔ I don't recall seeing frequent Helix timeouts and the higher the value here, the longer tests can hang the systems.

This point also doesn't seem like something we should mix into this PR. Suggest you file an issue for later.

Reply:
For the same reason we don't set a custom timeout to the expected running time on every job, consistency is good enough, this difference is just extra complexity.  We could set a timeout of 5 minutes on the majority of our jobs if we really care about low timeouts, the current situation is we probably have a bandaid that was slapped on to just barely not timeout on windows.  This type of thing is just a sign of an issue, but I'm just pointing it out since you moved it so it stood out, we certainly don't have to fix it now

Reply:
After offline discussion, I'm leaving this to you @HaoK. Your consistency argument convinced me and if we can _lower_ the timeout, all the better.